File: Searching-Algorithms/Linear_search/Python/linear_search.py
# Program to implement linear search 
count=0
N=int(input("Enter the number of elements you want to enter in an array : "))
arr=[N]
print("Enter {} elements : ".format(N))
for i in range(0,N):
	num=int(input())
	arr.append(num)

search=int(input("Enter the element you want to search : "))
for i in range(0,N):
	if(search==arr[i]):
		print("{} is present in the arrray.\n".format(search))
		break
	count+=1

if(count==N):
	print("{} is not present in an array.\n".format(search))

File: Searching-Algorithms/Linear_search/C/linear_search.c
/*
  Program to implement linear search
*/
#include<stdio.h>
int main()
{
	int N,T,search,count=0;
	printf("Enter the number of elements you want to enter in an array : ");
	scanf("%d",&N);
	int arr[N];
	printf("Enter %d elements : ",N);
	for(int i=0;i<N;i++)
	{
		scanf("%d",&arr[i]);
	}
	printf("Enter the element you want to search : ");
	scanf("%d",&search);
	for(int i=0;i<N;i++)
	{
		if(search==arr[i])
		{
			printf("%d is present in the arrray.\n",search);
			break;
		}
		count++;
	}
	if(count==N)
	{
		printf("%d is not present in an array.\n",search);
	}
	return 0;
}
